src/Homeworks/Homework5_1/Routers/IGraph.cs

```diff
@@ -0,0 +1,10 @@
+namespace Routers;
+
+public interface IGraph
+{
+    int Size { get; }
+
+    void AddEdge(int firstVertex, int secondVertex, int len);
+
+    (int, int)[] GetNeighbours(int vertex);
```

Comment:
Всё равно по стайлгайду лучше модификаторы видимости в интерфейсе явно прописать. А параметры кортежей проименовать.
Чтобы получилось как-то так:
```suggestion
    public (int Vertex, int Neighbour)[] GetNeighbours(int vertex);
```

Reply:
Не проименованы